### PR TITLE
docs: Adding Doug Parker to contributors list.

### PR DESCRIPTION
**Pull request by dgp1130**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

I am not in the contributors list.

Issue Number: N/A

## What is the new behavior?

I am now in the contributors list.

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

I listed Igor as my lead, though I'm honestly not sure what that means, so let me know if I should use someone else.

### CODE REVIEW

**Review by StephenFluin — CHANGES_REQUESTED** [review]
@dgp1130 I think you combined two unrelated changes into this PR. Can you revert the docs change so and submit it as a different PR so we can approve them independently?

**Comment by StephenFluin** [comment]
Also, if you can shrink the photo to around ~20kb, that would be great.

**Comment by dgp1130** [comment]
Just reverted it, apologies about that. I made a new branch but then forgot to switch to it before making the changes. 🤦‍♂

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -823,5 +823,14 @@
     "picture": "hafiz.jpg",
     "website": "https://github.com/ayazhafiz",
     "bio": "Ayaz is an undergraduate student with interests in free software."
+  },
+  "dgp1130": {
+    "name": "Doug Parker",
+    "picture": "dgp1130.jpg",
+    "twitter": "develwoutacause",
+    "website": "https://github.com/dgp1130",
+    "bio": "Doug is an overly-opionated software developer with a passion for making awesome developer tools. He is motivated primarily by his selfish desire to make his own life easier, but loves to share his tools and workflows with others to make everyone's lives easier.",
```
Comment (gkalpak) [inline aio/content/marketing/contributors.json]: > overly-opionated

Did you, by any chance, mean _"overly-op**in**ionated"_? :grin:

**Review by gkalpak — COMMENTED** [review]
Can you please remove the unrelated commits (`git revert` is not enough, because they will still show up in the main branches git history - you have to entirely remove the commits).

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (dgp1130) [reply aio/content/marketing/contributors.json]: Done, thanks for catching that.

**Comment by dgp1130** [comment]
Reduced the file size of the photo, dropped the two unrelated commits, and squashed them all together. I've used Git in a solo manner before, but am still learning how PRs work with it.

**Review by StephenFluin — APPROVED** [review]

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**